HIGHLIGHTS.md

```diff
@@ -7,3 +7,30 @@ What highlights can you come up with to share with the class?
 What do you really like? What can you find for fun facts related to any of these topics online?
 
 ## Cohort Highlights
+
+1. React Components and modular code- great way to break out our thoughts and our code. Makes it more easily readable for others (and ourselves a few weeks later!)
+
+2. HTTP debuggers files. Can easily tell if your code is working or not.
+
+3. JSX: HTML and Javascript living in harmony together. Example:
+const element = <h1>Hello, world!</h1>;
+
+4. Something I went out and learned on my own to add onto what we learned in class (basic JS) ... recursion. Example code snippit from the week we tackled some algorithms:
```

Comment:
This is excellent. This is a touch bit of code to understand. It's great to see this reference.